loc/views.py

```diff
@@ -190,19 +190,41 @@ def template_name_to_pdf_filename(template_name: str) -> str:
     return f'{filename.stem}.pdf'
 
 
-def render_english_to_string(request, template_name: str, context: Dict[str, Any]):
+def render_english_to_string(
+    request: Optional[HttpRequest],
+    template_name: str,
+    context: Dict[str, Any]
+):
     # For now, we always want to localize the letter of complaint in English.
     # Even if we don't translate the letter itself to other languages, some
     # templating functionality provided by Django (such as date formatting) will
     # take the current locale into account, and we don't want e.g. a letter to
     # have English paragraphs but Spanish dates. So we'll explicitly set
     # the locale here.
-    translation.activate('en')
+    with translation.override('en'):
+        return render_to_string(template_name, context=context, request=request)
```

Comment:
I realized that the old code activated the english locale but didn't re-activate the old locale.  This should do that, via [`django.utils.translation.override`](https://docs.djangoproject.com/en/3.0/ref/utils/#django.utils.translation.override).